app/javascript/mastodon/locales/defaultMessages.json

```diff
@@ -968,27 +968,6 @@
   {
     "descriptors": [
       {
-        "defaultMessage": "New report",
-        "id": "report.heading"
-      },
-      {
-        "defaultMessage": "Additional comments",
-        "id": "report.placeholder"
-      },
-      {
-        "defaultMessage": "Submit",
-        "id": "report.submit"
-      },
-      {
-        "defaultMessage": "Reporting",
-        "id": "report.target"
-      }
-    ],
-    "path": "app/javascript/mastodon/features/report/index.json"
-  },
```

Comment:
What happens if I run `build:development` after deleting the build directory once?

```console
$ rm -rf build
$ yarn build:development
$ yarn manage:translations
```